src/bindings/xfconf/elektra-xfconf.c

```diff
@@ -0,0 +1,73 @@
+#include "elektra-xfconf.h"
+#include "elektra-xfconf-util.h"
+
+GList * channel_list = NULL;
+KDB * elektraKdb = NULL;
```

Comment:
Are these global variables really necessary? Is this definitely only executed in a single-threaded context? Otherwise you'll need some thread synchronization...